back-end/controllers/snackController.js

```diff
@@ -0,0 +1,79 @@
+const express = require("express");
+const snacks = express.Router();
+const confirmHealth = require("../confirmHealth.js");
+const checkName = require('../validations/checkName.js');
+
+//QUERIES//
+const {
+  getAllSnacks,
+  getSnack,
+  createSnack,
+  updateSnack,
+  deleteSnack,
+} = require('../queries/snacks');
+
+snacks.get("/", async (req, res) => { 
+    const allSnacks = await getAllSnacks();
+    if(allSnacks[0]){
+        res.status(200).json({payload: allSnacks, success: true});
+    }else{
+        res.status(500).json({error: "server error"});
+    };
+});
+
+snacks.get("/:id", async (req, res) => {
+    const { id } = req.params;
+    const snack = await getSnack(id);
+    if(snack.id){
+        res.json({payload: snack, success: true});
+    }else{
+        res.status(404).json({payload: "not found", success: false});
+    };
+});
+
+snacks.post('/', async (req, res) => {
+    const { body } = req;
+
+    body.name = checkName(body);
```

Comment:
I'd lose the whitespace here as well - not really indicating a big change in logic